src/nvim/drawline.c

```diff
@@ -1041,7 +1041,9 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, int col_rows, b
              spellvars_T *spv, foldinfo_T foldinfo)
 {
   colnr_T vcol_prev = -1;             // "wlv.vcol" of previous character
-  ScreenGrid *grid = &wp->w_grid;     // grid specific to the window
+  GridView *grid = &wp->w_grid;     // grid specific to the window
+  const int cols = wp->w_view_width;
+  const int rows = wp->w_view_height;
```

Comment:
These two names are a bit too ambiguous. How about just using `view_width` and `view_height`?